README.md

```diff
@@ -26,10 +26,101 @@ var ECS = require('ecs-sdk');
 // import ECS object without services
 var ECS = require('ecs-sdk/global');
 
+// import individual service
+var INVENTORY = require('ecs-sdk/servers/inventory');
+
 // import individual service
 var TRACK = require('ecs-sdk/servers/track');
 
 // import logger
 var logger = require('ecs-sdk/lib/logger');
+```
+
+# Inventory
+- Stock Item Create
```

Comment:
sdk is not only for inventory, please move inventory docs to `/clients/inventory/readme.md`

and point the link to test case file

you can refer https://github.com/aws/aws-sdk-js